cpp/src/Slice/Parser.cpp

```diff
@@ -522,47 +522,88 @@ Slice::DocComment::parseFrom(const ContainedPtr& p, DocLinkFormatter linkFormatt
 
         if (parseNamedCommentLine(l, paramTag, name, lineText))
```

Comment:
It would be really nice to avoid naming anying `l`!

Reply:
Fixed. This was just the (unchanged) original name.
I renamed it to `line`.